x-pack/plugins/observability_solution/metrics_data_access/common/http_api/metrics_explorer.ts

```diff
@@ -91,6 +91,7 @@ export const afterKeyObjectRT = rt.record(rt.string, rt.union([rt.string, rt.nul
 
 export const metricsExplorerRequestBodyOptionalFieldsRT = rt.partial({
   groupBy: rt.union([groupByRT, rt.array(groupByRT)]),
+  groupInstance: rt.union([groupByRT, rt.array(groupByRT)]),
```

Comment:
same as earlier, I think `groupInstance` will always be an array.

Reply:
Answered above.